docs/website/docs/ml-frameworks/tensorflow-lite.md

```diff
@@ -1,14 +1,85 @@
 # TensorFlow Lite Integration
 
+IREE supports compiling and running pre-trained TensorFlow Lite (TFLite)
+models.  It converts a model to [TOSA MLIR](https://mlir.llvm.org/docs/Dialects/
+TOSA/), then compiles it into a VM module.
```

Comment:
Do these links work as written?

To break markdown lines at 80 characters, put any `[title](some-really-long-url-that-makes-this-go-beyond-80-characters)` links on a line by themselves

```suggestion
models.  It converts a model to
[TOSA MLIR](https://mlir.llvm.org/docs/Dialects/TOSA/),
then compiles it into a VM module.
```